keyboards/irene/keymaps/default/keymap.c

```diff
@@ -144,17 +176,30 @@ void matrix_init_user(void){
     lcd_drawCircle(64,32,7,WHITE);
     lcd_display();
     #endif
+   
+    timer1_init();
     return;
 }
 
 void matrix_scan_user(void){
-   lcd_clrscr();
-   lcd_puts("Irene's Keyboard");
-   lcd_gotoxy(0,2);
-   lcd_puts_p(PSTR("     ~(^o^~)"));
-#ifdef GRAPHICMODE
-   lcd_drawCircle(64,32,7,WHITE);
-   lcd_display();
-#endif
-   return;
+	if (idle_mode == 1) {
+		if ((TIFR1 & (1<<OCF1A)) != 0) {
```

Comment:
This is extremely confusing to read, timer register names are not common knowledge. Extensive documentation is necessary here.